### DISCUSSION TITLE
01/001-%E7%AC%AC%E4%B8%80%E7%AB%A0-%E8%AE%B8%E6%84%BF.html

### DISCUSSION THREAD

**Discussion by giscus** [liuli-moe/to-the-stars#28] [General]
01/001-%E7%AC%AC%E4%B8%80%E7%AB%A0-%E8%AE%B8%E6%84%BF.html
虽然出于伦理方面的考虑，我们必须针对许愿的后果和责任进行一些基本的指导，但此类指导必须要限制在符合人类社会整体利益的程度。教师不得透露 Incubator 系统在教学大纲以外的任何信息。这一限制将持续到学生超出契约年龄为止，通常认为是 20 岁。虽然对学生的关心是可贵的，但保持沉默是您作为一名公民不可推卸的责任。违反这一规定的教师将立即撤职并依法提起公诉……
https://tts.liuli.moe/01/001-%E7%AC%AC%E4%B8%80%E7%AB%A0-%E8%AE%B8%E6%84%BF.html

**Comment by name8102** [comment]
挑个错，原文中的Field Theory中文应该是场论而不是群论

**Reply by rxliuli** [reply]
感谢反馈，看翻译完全没留意到，已确认有 5 个章节受到影响

**Reply by rxliuli** [reply]
已修正，ref: <https://github.com/liuli-moe/to-the-stars/commit/c57d9de8247555e77f695b19a43b6649b3a096dd>

**Comment by xuelaSan** [comment]
我想,我们所缺乏的，不是技术与魔法少女，是相信奇迹的内心与向往星空的昂扬。或许有一天，那些渴望也能重新唤醒我们。

**Comment by 25C2232** [comment]
【一起去天国吧｜天国へ行こう/きくお feat. 初音ミク【オルゴール】-哔哩哔哩】 https://b23.tv/Zqf2B0g

嗯,很好聽,說不定可以用來做手書
